### DISCUSSION TITLE
Math Subscripting text

### DISCUSSION THREAD

**Discussion by toihr** [typst/typst#1261] [Q&A] [answered]
Hi everyone, I am very new to Typst and just found it over my most recent rage about LaTeX. 
There was something I was just wondering in Math mode. As a physicist sometimes it becomes necessary to subscript a symbol with Text like $\eta_{eff}$. What is the intended way to write this in Typst. I have so far only figured out that I could use a string aka `$eta_"eff"$`. I just wanted to ask if this is the intended way to subscript short texts or if there is another feature I was missing.

Kind regards
Tom

**Comment by laurmaedje** [comment, marked as answer]
The string is the way to go!
